cypress/integration/plugin_test/visualizer_spec.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" /> // Add and extra '/' to load the cypress auto-complete feature following npm i cypress
+import { VISUALIZER } from "../../support/constants"
+
+// DO NOT INTERCEPT CAPABILITIES WITH FIXTURES 
+describe("Visualizer Spec", () => {
+  beforeEach(() => {
+    cy.login();
+    cy.setReleaseTag();
+    window.localStorage.setItem("tab", 0);
+    const token = Cypress.env('token');
+    cy.request(`http://localhost:9081/api/user/token?token=${token}`);
```

Comment:
@Abhishek-kumar09 Can we remove the port hardcoding here?

Reply:
Good eye. Checking the changes locally.